pkg/kubectl/cmd/patch.go

```diff
@@ -67,7 +67,7 @@ func NewCmdPatch(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 			cmdutil.CheckErr(err)
 		},
 	}
-	cmd.Flags().StringP("patch", "p", "", "The patch to be applied to the resource JSON file.")
+	cmd.Flags().StringP("patch", "p", "", "The patch to be applied to the resource JSON file, or - to use files on STDIN as the patch source.")
```

Comment:
Agree, I need to come back to this.  I was using it as a less
opinionated apply for specific scenarios.  I need to validate it
actually improves over apply in the scenario. I tried to use.